openedx/core/djangoapps/profile_images/images.py

```diff
@@ -0,0 +1,132 @@
+"""
+Image file manipulation functions related to profile images.
+"""
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from django.utils.translation import ugettext as _, ugettext_noop as _noop
+from PIL import Image
+
+from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_storage
+
+
+FILE_UPLOAD_TOO_LARGE = _noop('Maximum file size exceeded.')
+FILE_UPLOAD_TOO_SMALL = _noop('Minimum file size not met.')
```

Comment:
The file must be at least 10 KB in size. (Substitute correct number for "10 KB")